src/desktop/apps/artist2/client.js

```diff
@@ -20,12 +20,13 @@ buildClientApp({
   routes,
   context: {
     user: sd.CURRENT_USER,
+    mediator,
```

Comment:
Oof 😦

Reply:
Refactored everything for this specific reason and then didn't update the actual app.